docs/_index.md

```diff
@@ -22,27 +42,94 @@ const repository = new nuage.aws.Repository("repository-resourcename",{
 
 const lambdaContainer = new nuage.aws.ContainerFunction("myfunction",{
     name:"lambda-function",
+    description:"Nuage AWS ContainerFunction resource.",
     dockerfile:"./Dockerfile",
-    repositoryUrl:repository.url
+    context:"./lambda/",    
+    repositoryUrl:repository.url,
+    architecture:"x86_64",
+    memorySize:512,
+    timeout:30,
+    environment:{DUMMY:"VARIABLE"},
+    keepWarm:true,
+    url:true,
+    logRetentionInDays:90
 });
 ```
- 
+
 {{% /choosable %}}
 {{% choosable language python %}}
 
 ```python
-import pulumi_nuage
+import pulumi_nuage as nuage
 
-repository = pulumi_nuage.aws.Repository(
+repository = nuage.aws.Repository(
     "repository-resourcename",
     name="repository",
     expire_in_days=30,
 )
 
-container_function = pulumi_nuage.aws.ContainerFunction("myfunction",
+container_function = nuage.aws.ContainerFunction("myfunction",
     name="lambda-function",
-    repository_url=repository.url,
+    description="Nuage AWS ContainerFunction resource.",
     dockerfile="./lambda/Dockerfile.lambda",
+    context="./lambda/",
+    repository_url=repository.url,    
+    architecture="x86_64",
+    memory_size=512,
+    timeout=30,
+    environment={"DUMMY":"VARIABLE"},
+    keep_warm=True,
+    url=True,
+    log_retention_in_days=90
+)
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+
+### ServerlessDatabase
+
+The ServerlessDatabase component is an effective solution for creating serverless databases using Amazon Relational Database Service (RDS) Aurora databases. It streamlines the process by automatically creating the necessary components, including a subnet group, security group, security group rules, an RDS cluster and data API if requested.
```

Comment:
Maybe we should forget about the data API now that Aurora sls v2 has removed it... ?

Reply:
We could also say that our component generates a random password securely managed by Pulumi